compliance/fnms/FNMS_Licenses_At_Risk/README.md

```diff
@@ -0,0 +1,64 @@
+## FlexNet Manager Licenses At Risk.
+
+### What it does
+This policy will look up the Licenses that are at risk, meaning that there is probably not enough entitlements for the consumed applications.
+The policy is a report only policy, no action is taken during the Policy Escalation
+
+The report / Mail output looks like this:
+![Alt text][emailoutput]
+
+Current limitations:
+- This only works with the FNMS Cloud
+- Output is limited to max 100.000 rows.
```

Comment:
this might be a localization difference, should the limit be 100.000 or 100,000 or maybe just 100000.  in the US locale 100.000 is really 100.0.  Since this is in english I think it should ready 100,000 or 100000.